### DISCUSSION TITLE
Math dataset postprocess not good

### DISCUSSION THREAD

**Discussion by RanchiZhao** [open-compass/opencompass#491] [Q&A]
I've noticed that the math dataset isn't as robust as I'd hoped. Despite the numerous post-processing functions available, they don't seem to perfectly handle the model outputs.

**Comment by Leymore** [comment]
Totally agreed! Would you like to help improve this postprocess?

**Reply by RanchiZhao** [reply]
I am trying to do this, although it is especially hard to do this for base models.
